index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <div class="nav-bar">
+            <div class="logo">Charles</div>
+            <img src="images/Union.png" alt="Menu Bar">
+        </div>
+        <div class="content">
```

Comment:
- [ ] This is your headline section so it would be better to use the `<section>` tag instead of the` <div> tag in order to improve SEO also `<section>` tag is made for sections

Reply:
done